### DISCUSSION TITLE
Is it safe to migrate from Trilium to TriliumNext?

### DISCUSSION THREAD

**Discussion by welrbraga** [TriliumNext/Notes#394] [Q&A / Support] [answered]
Hi guys,

I stucked my install in Trilium 0.63.7 and DB 228 with the oficial Docker image while following and waiting anxiously that transition to TriliumNext. After a little time without open GitHub I was surprised in see the new release of TriliumNext 0.9.0.

More. I noted that TriliumNext Docker Images are tagged with their digest SHA256, instead of the app version, like Trillium (0.63.7, or 0.63-latest etc).

So, I have a simple doubt:  Is it safe to upgrade my production install now, or is better wait a little more?

Thanks

**Comment by meichthys** [comment, marked as answer]
It is safe to upgrade. The regular tags are there, but they just got buried a little bit. You can find the latest release here.
https://hub.docker.com/r/triliumnext/notes/tags?page_size=&ordering=&name=latest

**Reply by welrbraga** [reply]
Thanks @meichthys ,

You were quicker than me. I just found the tag v0.90.4 and return to edit the question, but now it's unnecessary.

I will prepare my upgrade. 

Thanks and thanks

**Comment by Talisker69** [comment]
Hi,
Just migrate today with the new docker image every things are ok
Thanks to continue this project ....
Enjoy
